### PR TITLE
Update to go 1.15

### PR DESCRIPTION
**Pull request by hachi8833**
* Update .travis.yml for testing on Go 1.14 and 1.15
  * Remove unnecessary `GO111MODULE` env
* Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,18 +26,30 @@ The expected use case for Goby would be backend development. With this goal, it
...
-- [Features](#major-features)
-- [Installation](#installation)
-- [Usage](#verifying-goby-installation)
-- [Sample codes](#sample-codes)
-- [Documentation](https://goby-lang.org/docs/introduction.html)
-- [Joining to Goby](#joining-to-goby)
-- [Maintainers](#maintainers)
-- [Supporters](#supporters)
-- [References](#references)
+- [!Goby](#img-srchttpiimgurcomelgazrnpng3-altgoby)
```
Comment (st0012) [inline README.md:29]: not sure this is working as expected?
截圖 2020-10-19 下午4 21 38

Comment (hachi8833) [reply README.md:29]: Oh, I should revert the part!
Just a moment

**Comment by hachi8833** [comment]
It seems that my VSCode config is a bit glitched😅
I used Sublime instead and worked fine

**Review by st0012 — APPROVED** [review]
Looks good, thanks 👍

**Pull request merged**